packages/components/pie-radio-group/src/defs.ts

```diff
@@ -1,3 +1,38 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface RadioGroupProps {}
+import { type ComponentDefaultProps } from '@justeattakeaway/pie-webc-core';
+
+export interface RadioGroupProps {
+    /**
+    * The name associated with the group.
+    */
+    name?: string;
+
+    /**
+     * Inline (horizontal) positioning of radio items
+     */
+    isInline?: boolean;
+
+    /**
+     * Same as the HTML disabled attribute - indicates whether or not the radio group is disabled.
+     */
+    disabled?: boolean;
+
+    /**
+     * The value of the radio group (used as a key/value pair in HTML forms with `name`).
+     */
+    value?: string;
+}
+
+/**
+ * Event name for when radio group becomes disabled.
+ *
+ * @constant
+ */
+export const ON_RADIO_GROUP_DISABLED = 'pie-radio-group-disabled';
+
+export type DefaultProps = ComponentDefaultProps<RadioGroupProps, 'value' | 'disabled' | 'isInline'>;
```

Comment:
Small suggestion, but if you instead do this (I wrote it by hand so probably best not to commit it directly from the PR)
```suggestion
export type DefaultProps = ComponentDefaultProps<RadioGroupProps, keyof Omit<RadioGroupProps, 'name'>>;
```
then it functions the same but if a new prop is added in future then by default the type will have an error. This forces the author to either add a default value below or add the prop name to the exclusion list to make the error go away.